<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doctorEditview</class>
 <widget class="QWidget" name="doctorEditview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>医生编辑</string>
  </property>
  <widget class="QPushButton" name="btSave">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCancle">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>165</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>医生工号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="IDEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>165</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>医生证书号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="bookEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>156</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>医生姓名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>156</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>医生年龄</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ageEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>102</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>医生性别</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="sexBox">
      <item>
       <property name="text">
        <string>男</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>女</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
